parquet/src/test/scala/magnolify/parquet/AvroParquetSuite.scala

```diff
@@ -162,11 +162,14 @@ class AvroParquetSuite extends MagnolifySuite {
   {
     import magnolify.avro.logical.bigquery._
     // Precision = number of digits, so 5 means -99999 to 99999
-    val nines = math.pow(10, 38).toLong - 1
+    val precision = 38
```

Comment:
Apparently you only changed the code style of test suite. What is the test case when it fails without the fix in `Schema.scala`?